src/components/ha-cover-controls.html

```diff
@@ -0,0 +1,60 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<link rel="import" href="../util/cover-model.html">
+
+<dom-module id="ha-cover-controls">
+  <template>
+    <style>
+      .state {
+        text-align: right;
```

Comment:
This should be part of `state-card-cover.html` as it's about how the component should be rendered.